bootstrap/kubeadm/config/default/deployment_remove_selectors.yaml

```diff
@@ -0,0 +1,4 @@
+- op: remove
+  path: "/spec/selector/matchLabels/clusterctl.cluster.x-k8s.io"
+- op: remove
+  path: "/spec/selector/matchLabels/clusterctl.cluster.x-k8s.io~1provider"
```

Comment:
Why do we need to remove this?

Reply:
If I got this right 
- commonLabels applies labels to all the objects and to all the selectors
- this removes the clusterctl labels from the selectors

Reply:
`~1` appears to be an escape sequence for `/`

Reply:
Seems a little ugly, but oh well :D

Reply:
Yeah...Unfortunately there was no other way I could find to avoid that special character sequence. 😐